dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -221,7 +221,7 @@ public AbstractInterfaceConfig(ModuleModel moduleModel) {
     /**
      * The url of the reference service
      */
-    protected final transient List<URL> urls = new ArrayList<URL>();
+    protected final transient List<URL> urls = new ArrayList<>();
```

Comment:
**Suggestion**: Good use of the diamond operator to simplify the instantiation of generic classes. This keeps the code cleaner and easier to read. <br> **Code Suggestion**: 
 ```
 No further code suggestion required, the change is appropriate.
 ```